### PR TITLE
add CtrlSFToggle

### PR DESCRIPTION
**Pull request by zhyu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +106,8 @@ vmap     <C-F>F <Plug>CtrlSFVwordExec
 nmap     <C-F>n <Plug>CtrlSFCwordPath
 nmap     <C-F>p <Plug>CtrlSFPwordPath
 nnoremap <C-F>o :CtrlSFOpen<CR>
+nnoremap <silent><C-F>t :CtrlSFToggle<cr>
+imap <C-F>t <Esc>:CtrlSFToggle<cr>
```
Comment (dyng) [inline README.md]: Please align these two lines, and I think `<silent>` is unnecessary. (It's not something bad but better to delete it for... um, alignment)

File: autoload/ctrlsf.vim [hunk 2]
```diff
@@ -232,3 +232,16 @@ endf
 func! ctrlsf#ClearSelectedLine() abort
     call ctrlsf#hl#ClearSelectedLine()
 endf
+
+" Toggle()
+"
+func! ctrlsf#Toggle() abort
+    let ctrlsfwinnr = bufwinnr("__CtrlSF__")
```
Comment (dyng) [inline autoload/ctrlsf.vim]: I think you should use `ctrlsf#win#FindMainWindow()`.

Comment (zhyu) [inline autoload/ctrlsf.vim]: Oh, you're right!

File: autoload/ctrlsf.vim [hunk 3]
```diff
@@ -232,3 +232,16 @@ endf
...
+    if ctrlsfwinnr != -1
+        call ctrlsf#preview#ClosePreviewWindow()
+        call ctrlsf#win#CloseMainWindow()
```
Comment (dyng) [inline autoload/ctrlsf.vim]: Maybe `ctrlsf#Quit()` is better. I can understand your efforts of keeping using low-level methods, but I think it's ok using `ctrlsf#Quit()` here, that is exactly what we are doing.

File: autoload/ctrlsf.vim [hunk 4]
```diff
@@ -232,3 +232,16 @@ endf
...
+        return
+    endif
+
+    call ctrlsf#win#OpenMainWindow()
```
Comment (dyng) [inline autoload/ctrlsf.vim]: Same as above comment. `ctrlsf#Open()` may be better.

**Pull request merged**